source/api/presentation/3.0/index.md

```diff
@@ -1490,7 +1484,23 @@ Access-Control-Allow-Origin: *
 
 Responses _SHOULD_ be compressed by the server as there are significant performance gains to be made for very repetitive data structures.
 
-Recipes for enabling CORS and the conditional Content-type header are provided in the [Apache HTTP Server Implementation Notes][apache-notes].
+Recipes for enabling CORS and the conditional Content-Type header are provided in the [Apache HTTP Server Implementation Notes][apache-notes].
+
+### 6.3 URI Recommendations
+
+While any URI is technically acceptable for any of the resources, there are several best practices for designing the URIs for the resources.
+
+* The URI _SHOULD_ use the HTTPS scheme, not HTTP.
+* Consider the hostname for your resources, such that it could redirect to separate systems if needed. For example: `https://data.example.org/`
+* Use a consistent prefix path for IIIF resources, such as `https://data.example.org/iiif/`
+* Use a natural identifier for the resource in its management system in the URI, such as `https://data.example.org/iiif/849/` for object number 849
+* Use the type of the resource in the URI, such as `https://data.example.org/iiif/849/manifest/`
+* Don't use `.json` or `.jsonld` on the end of the URI unless you need to due to implementation requirements
+* Remember to encode special characters
+* Consider future expansion of the data, and related services, such as `https://data.example.org/iiif/849/manifest/search/` for a [IIIF search service][search-api] over the object's annotations
+* Do not use query parameters or fragments as part of the URI
+* When dereferenced, the URI _SHOULD_ return the JSON-LD description of the resource
+* Once published, they _SHOULD_ be as persistent and unchanging as possible
```

Comment:
Discussion in Eds meeting is that the normative ones should probably stay here:

  * The URI _SHOULD_ use the HTTPS scheme, not HTTP.
  * Do not use query parameters or fragments as part of the URI --- make stronger _SHOULD NOT_ ?
  * When dereferenced, the URI _SHOULD_ return the JSON-LD description of the resource -- is this actually a _MUST_ in order for IIIF to work?
  * Once published, they _SHOULD_ be as persistent and unchanging as possible

And then link off to patterns in an implementation note. Within those, not sure that "Use a natural identifier for the resource in its management system in the URI, such as `https://data.example.org/iiif/849/` for object number 849" is good as this may actually get in the way of persistence if system is changed

Reply:
That seems good. I was struggling to fulfill the requests in the issue and still keep it in the spec. The implementation note would be part of a larger document, I imagine?

Reply:
I imagine we'll find more to add, but I think starting a prezi3 implementation notes document with this would seem good for now